src/UseCases/CancelDonation/CancelDonationUseCase.php

```diff
@@ -59,17 +59,31 @@ public function cancelDonation( CancelDonationRequest $cancellationRequest ): Ca
 
 		$this->donationLogger->log( $donation->getId(), self::LOG_MESSAGE_FOR_BACKEND );
 
-		$this->sendConfirmationEmail( $donation );
+		try {
+			$this->sendConfirmationEmail( $donation );
+		}
+		catch ( \RuntimeException $ex ) {
+			return new CancelDonationResponse(
```

Comment:
My reasoning was that there is just one caller. Same goes for the success case. So yeah, kinda inconsistent. Guess extracting it is a bit nicer.